store/node.go

```diff
@@ -212,13 +212,14 @@ func (node Node) accumulate() (res sdk.Int) {
 	return
 }
 
+//nolint:revive
```

Comment:
what is this linter?

Reply:
revive replaced.... golint -- it does a whole lot of stuff:

```
Fast, configurable, extensible, flexible, and beautiful linter for Go. Drop-in replacement of golint. Revive provides a framework for development of custom rules, and lets you define a strict preset for enhancing your development & code review processes.
```